t4c/t4c_cli/backup.py

```diff
@@ -206,46 +176,41 @@ def copy_exported_files_into_git_repo(
     log.info("Finished copying files...")
 
 
-def create_t4c_project_dir(t4c_config: config.T4CConfig):
-    project_dir = pathlib.Path(t4c_config.project_dir_path)
+def create_t4c_project_dir():
+    project_dir = pathlib.Path(config.config.t4c.project_dir_path)
     project_dir.mkdir(exist_ok=True)
 
 
 @click.command()
 def backup():
-    general_config: config.GeneralConfig = config.GeneralConfig(
-        error_prefix="Import of model from TeamForCapella server failed"
-    )
-    t4c_config: config.T4CConfig = general_config.t4c_config
-    git_config: config.GitConfig = general_config.git_config
+    git_config: config.GitConfig = config.config.git
+    file_handler = config.config.file_handler
 
-    create_t4c_project_dir(t4c_config)
+    create_t4c_project_dir()
 
     last_backup_commit_datetime = None
-    if general_config.file_handler == "GIT":
-        git.clone_git_repository_to_git_dir_path(general_config)
+    if file_handler == config.FileHandler.GIT:
+        util_git.clone_git_repository_to_git_dir_path()
         last_backup_commit_datetime = (
-            git.find_last_commit_timestamp_by_text_search(
+            util_git.find_last_commit_timestamp_by_text_search(
                 git_config=git_config, grep_arg="Backup"
             )
         )
 
-    run_importer_script(general_config, last_backup_commit_datetime)
-    unzip_exported_files(t4c_config)
+    run_importer_script(last_backup_commit_datetime)
+    unzip_exported_files()
 
-    if general_config.file_handler == "GIT":
-        copy_exported_files_into_git_repo(general_config)
+    last_t4c_commit_datetime = None
```

Comment:
Nit: This line is not necessary here because the `last_t4c_commit_datetime' is only assigned and used within the following `if' block.